src/sql/base/browser/ui/editableDropdown/browser/dropdown.ts

```diff
@@ -316,6 +316,8 @@ export class Dropdown extends Disposable implements IListVirtualDelegate<string>
 	private _hideList(): void {
 		this.contextViewService.hideContextView();
 		this._inputContainer.setAttribute('aria-expanded', 'false');
+		// Show error for input box in case the user closed the dropdown without selecting anything, like by hitting Escape
+		this.input.validate();
```

Comment:
I moved validate here because we were actually hiding the error message immediately after showing it, since we call hideContextView above and the error message is actually a context view. Unfortunately, that also means we can't show an error _and_ the dropdown list at the same time, since only one context view can be displayed at a time.